src/toolchain/export-web.md

```diff
@@ -109,14 +112,129 @@ cargo +nightly build -Zbuild-std --target wasm32-unknown-emscripten
 ```
 
 
+## Thread support (Godot 4.3 or later)
+
+The above settings assume that multi-threading support is always needed for your extension. However, starting with Godot 4.3, when the end user
+exports a game to the web, Godot includes an option to disable Thread support in the web export menu
+(see the image in the ["Godot editor setup" section](#godot-editor-setup)), with the goal of having the exported game run in more environments,
+including older browsers, as well as webservers without Cross-Origin Isolation support.
+
+If the end user disables Thread support using that new option, however, your extension, without any changes, would break.
```

Comment:
Simplified flow + conjunctive + italics:
```suggestion
If the end user disabled _Thread support_ without applying any other changes, your extension would break.
```

Reply:
I changed it a bit further to indicate that the necessary changes should come from the extension developer.